### PR TITLE
feat(web): add docs for step types in workflow floating menu

### PR DESCRIPTION
**Pull request by davidsoderberg**

### CODE REVIEW

File: apps/web/src/ee/translations/pages/TranslationGroupsPage.tsx [hunk 1]
```diff
@@ -45,9 +47,12 @@ export const TranslationGroupsPage = () => {
...
     <PageContainer>
       <Group position="apart" mx={24} mt={24} mb={16} align="center">
-        <Title size={2} data-test-id="translation-title">
-          Translations
-        </Title>
+        <Flex align="center" gap="75">
+          <Title size={2} data-test-id="translation-title">
+            Translations
+          </Title>
+          <DocsButton />
```
Comment (davidsoderberg) [inline apps/web/src/ee/translations/pages/TranslationGroupsPage.tsx:54]: This one was not added during when I created the docs button but now when ee web is moved here this is easy and possible.

File: apps/web/src/studio/components/workflows/node-view/WorkflowFloatingMenu.tsx [hunk 2]
```diff
@@ -14,63 +14,74 @@ import {
...
-    alert('This should open docs for ' + stepType);
+  const [docsOpen, setDocsOpen] = useState<boolean>(false);
+  const [path, setPath] = useState<string>('');
+
+  const toggle = () => {
+    setDocsOpen((prevOpen) => !prevOpen);
+  };
+
+  const handleClick = (pathToSet: string) => () => {
+    setPath('echo/steps/' + pathToSet);
```
Comment (davidsoderberg) [inline apps/web/src/studio/components/workflows/node-view/WorkflowFloatingMenu.tsx:31]: To make it easy for use to switch when we have changed echo in our docs path

Comment (SokratisVidros) [reply apps/web/src/studio/components/workflows/node-view/WorkflowFloatingMenu.tsx:31]: What does this `setPath` do? Is it a navigation?

Comment (davidsoderberg) [reply apps/web/src/studio/components/workflows/node-view/WorkflowFloatingMenu.tsx:31]: It sets the path of what docs the modal should render.

**Review by SokratisVidros — APPROVED** [review]

**Pull request merged**